tests/bluetooth/host/buf/bt_buf_get_cmd_complete/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ project(bluetooth_bt_buf_get_cmd_complete)
 
 find_package(Zephyr COMPONENTS unittest HINTS $ENV{ZEPHYR_BASE})
 
-include(${ZEPHYR_BASE}/tests/bluetooth/host/host_mocks.cmake)
-include(${ZEPHYR_BASE}/tests/bluetooth/host/buf/mocks.cmake)
+add_subdirectory(${ZEPHYR_BASE}/tests/bluetooth/host/    host_mocks)
+add_subdirectory(${ZEPHYR_BASE}/tests/bluetooth/host/buf mocks)
```

Comment:
Why there is an extra space? It doesn't compile if it is removed

Reply:
the `add_subdirectory()` takes two args, like this:
`add_subdirectory(<arg1> <arg2>)`

any extra space >1 are for aligning `<arg2>` over each other for readability.